src/index.js

```diff
@@ -417,9 +417,9 @@ class Masq {
    */
   async get (key) {
     const db = this._getDB()
-    const nodes = await db.getAsync(key)
-    if (!nodes.length) return nodes[0]
-    return nodes[0].value
+    const node = await db.getAsync(key)
+    if (!node) return undefined
```

Comment:
according to https://github.com/mafintosh/hyperdb/blob/master/test/basic.js#L38, we should return `null` I think

Reply:
Ca you verify that node is null ? If so, return null too

Reply:
yes with firstNode: true it returns null if the key does not exist.